### PR TITLE
Add rubocop

### PR DESCRIPTION
**Pull request by giovannism20**
Solving problems from PR #32 and also will be possible to merge PR #31 (which closes issue #26). Closes issue #15

### CODE REVIEW

**Comment by keshavbiswa** [comment]
This is great! But let's try and keep it simple, let's try with a very simple rubocop.yml config file and then we can progress incrementally. For now we don't need other gems, just rubocop.

File: Gemfile [hunk 1]
```diff
@@ -6,10 +6,16 @@ source "https://rubygems.org"
...
 group :test do
   gem "simplecov", "~> 0.22.0", require: false
 end
+
+group :development, :test do
+  gem "rubocop", require: false
+  gem "rubocop-performance", require: false
+  gem "rubocop-rails", require: false
+  gem "rubocop-rspec", require: false
+end
```
Comment (keshavbiswa) [inline Gemfile:21]: Let's remove this for now, and only focus on rubocop gem with a minimal rubocop.yml config file.

**Review by keshavbiswa — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (giovannism20) [reply Gemfile]: Sure @keshavbiswa. Updated!

**Review by keshavbiswa — CHANGES_REQUESTED** [review]
Thanks for making the change. One final thing we might need. Could you delete the existing `rubocop.yml` and run 
```shell
rubocop --auto-gen-config
```
This will generate a proper rubocop config with exclusions and tests should pass.

**Review by keshavbiswa — CHANGES_REQUESTED** [review]
There are 87 file changes which I think shouldn't be here if we have the rubocop and rubocop_todo.yml in place, we can get rid of all of these file changes, this way tests will pass and we can look into the rubocop_todo in other issues.

**Comment by keshavbiswa** [comment]
Okay I think the problem here still is that now that you've removed the changed files, we need to run rubocop --auto-gen-config again to generate the rubocop_todo.yml file.

After that run rubocop locally to ensure it passes?

**Review by keshavbiswa — CHANGES_REQUESTED** [review]
Okay I think the problem here still is that now that you've removed the changed files, we need to run rubocop --auto-gen-config again to generate the rubocop_todo.yml file.

After that run rubocop locally to ensure it passes?

**Comment by giovannism20** [comment]
Rubocop is passing locally @keshavbiswa

File: spec/generators/install_generator_spec.rb [hunk 2]
```diff
@@ -149,7 +149,7 @@
     end
 
     it "sorts models by dependency" do
-      expect(content["models"].keys).to eq ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
+      expect(content["models"].keys).to eq ["post", "simple_model", "user", "comment", "game_room", "game_room_user", "post_metadatum", "review"]
```
Comment (keshavbiswa) [inline spec/generators/install_generator_spec.rb:152]: This change is causing the tests to fail. Can you revert this?

**Review by keshavbiswa — CHANGES_REQUESTED** [review]

File: spec/generators/install_generator_spec.rb (hunk 2 above)
Comment (giovannism20) [reply spec/generators/install_generator_spec.rb]: Reverted but when I run this test I'm getting this problem:
```Failures:

  1) Seedie::Generators::InstallGenerator#model_configuration sorts models by dependency
     Failure/Error: expect(content["models"].keys).to eq expect(content["models"].keys).to eq ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
     
       expected: ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
            got: ["post", "simple_model", "user", "comment", "game_room", "game_room_user", "post_metadatum", "review"]
     
       (compared using ==)
     # ./spec/generators/install_generator_spec.rb:152:in `block (3 levels) in <top (required)>'

Finished in 1.61 seconds (files took 0.93248 seconds to load)
160 examples, 1 failure

Failed examples:

rspec ./spec/generators/install_generator_spec.rb:151 # Seedie::Generators::InstallGenerator#model_configuration sorts models by dependency```

File: spec/generators/install_generator_spec.rb [hunk 3]
```diff
@@ -149,7 +149,7 @@
     end
 
     it "sorts models by dependency" do
-      expect(content["models"].keys).to eq ["post", "simple_model", "user", "comment", "game_room", "game_room_user", "post_metadatum", "review"]
+      expect(content["models"].keys).to eq expect(content["models"].keys).to eq ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
```
Comment (keshavbiswa) [inline spec/generators/install_generator_spec.rb:152]:
```suggestion
expect(content["models"].keys).to eq ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
```

**Review by keshavbiswa — CHANGES_REQUESTED** [review]

File: spec/generators/install_generator_spec.rb (hunk 2 above)
Comment (keshavbiswa) [reply spec/generators/install_generator_spec.rb]: Interesting, maybe your db has a different schema, will have to take a deeper look.

**Comment by keshavbiswa** [comment]
@giovannism20 I fixed the failing test, can you fix the indentation problem with rubocop?

**Comment by giovannism20** [comment]
Done @keshavbiswa

**Comment by giovannism20** [comment]
But about tests I'm still getting the same problem even after your modification

**Comment by keshavbiswa** [comment]
> But about tests I'm still getting the same problem even after your modification

We'll have to take a look at your db setup.

**Comment by keshavbiswa** [comment]
@giovannism20 The build passed, thanks for all the hardwork 👍

**Review by keshavbiswa — APPROVED** [review]
LGTM!

**Comment by giovannism20** [comment]
I've runned setup ```setup``` and then run ```bundle exec rspec```

**Pull request merged**